code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -441,3 +441,15 @@
 		M.emote(pick("twitch","drool","moan"))
 	..()
 
+/datum/reagent/drug/genderfluid
+	name = "Gender Fluid"
+	id = "genderfluid"
+	description = "A curious fluid that causes a patient's gender to become wildly inconsistent!"
+	reagent_state = LIQUID
+	color = "#9F60EB"
+	taste_description = "gender politics"
+	addiction_threshold = 10
```

Comment:
this currently does nothing beside giving the user a negative moodlet and craving messages. Intended?

Reply:
Non-harmful addiction is intended, yes.

Reply:
>
> self-discovery is one hell of an addicting drug.

Well, I'd like to differ but ok, go ahead.